doc/quick-start-user-guide.md

```diff
@@ -200,3 +200,53 @@ Verify that `glusterfsd` process is running on both nodes.
 * Issues with 2 node clusters
   * Restarting glusterd2 does not restore the cluster
   * Peer remove doesn't work
+
+### Solutions to common issues
+
+  >  Note: These are hacks or temporary solutions to the problems.
+
+* If glusterd service fails with error: "failed to start embedded store"
+
+  Sample output:
+  ```log
+  ERRO[2018-06-04 06:05:41.020313] failed to start embedded store                error="dial tcp {IP}: connect: connection refused" source="[embed.go:36:store.newEmbedStore]"
+  ```
+  To solve this you will be required to clean up glusterd2 [working directory](https://github.com/gluster/glusterd2/blob/master/doc/quick-start-user-guide.md#running-glusterd2).
+  The path to default directory used by glusterd2 is ["/var/lib/glusterd/"](https://github.com/gluster/glusterd2/blob/master/glusterd2.toml.example#L1), if using custom config file then please provide working directory path instead of "/var/lib/glusterd/"
+
+  ```sh
+  # rm /var/lib/glusterd/*
```

Comment:
do we need to delete the complete glusterd2 directory? it will delete the old ETCD data which is stored in ETCD folder. ETCD config will be stored in store.toml (I think clearing this should be enough). If store.toml is empty glusterd2 should regenerate the store.toml based on the configuration

Reply:
This comment is still not addressed?

Reply:
@Madhu-1 I will provide a warning, and will send a new PR after testing whether its suffiecient to delete store.toml.